webdriver-spec.html

```diff
@@ -2843,7 +2843,7 @@
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+ <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
```

Comment:
For the _Back_ command, we only need to check the top-level browsing context since it cannot be executed if the window is missing.